lib/source.c

```diff
@@ -101,6 +103,7 @@ tlog_grc
 tlog_source_read(struct tlog_source *source, struct tlog_pkt *pkt)
 {
     tlog_grc grc;
+    struct timespec diff;
```

Comment:
Could you please wrap this line in `#ifndef NDEBUG`? Otherwise the compiler will complain this variable is unused when built with `NDEBUG`.

Reply:
Oh I already put this line in `#ifndef NDEBUG` in commit issue, and you told me put this line on the top. It makes me little confused about it because I know the compiler will complain this when variable is unused when built with `NDEBUG`. I will wrap this lane and 2 #include in `NDEBUG`.